src/Logary/Configuration/Ticker.fs

```diff
@@ -28,11 +29,11 @@ type Ticker<'state,'t,'r> (initialState:'state) =
   member this.Ticked = tickCh :> Alt<_>
   member this.Tick () = tickCh *<- ()
 
-  member this.TickEvery timespan =
+  member this.TickEvery (duration: Duration) =
     let cancellation = Cancellation.create ()
     let rec loop () =
       Alt.choose [
-        timeOut timespan ^=> fun _ ->
+        timeOut (duration.ToTimeSpan()) ^=> fun _ ->
```

Comment:
You need to guard the boundaries here, since e.g. Duration.Zero and Duration.MaxValue (??) may cause runtime exceptions.

Reply:
will do it later